x-pack/plugin/src/yamlRestTest/resources/rest-api-spec/test/esql/140_metadata.yml

```diff
@@ -0,0 +1,148 @@
+setup:
+  - requires:
+      test_runner_features: [capabilities]
+      capabilities:
+        - method: POST
+          path: /_query
+          parameters: [method, path, parameters, capabilities]
+          capabilities: [metadata_fields, metadata_field_ignored]
+      reason: "Ignored metadata field capability required"
+
+  - do:
+      indices.create:
+        index: test
+        body:
+          mappings:
+            properties:
+              integer:
+                type: integer
+                ignore_malformed: true
+              keyword:
+                type: keyword
+                ignore_above: 3
+              case:
+                type: keyword
+
+  - do:
+      bulk:
+        index: test
+        refresh: true
+        body:
+          - { index: { } }
+          - { case: "ok", integer: 10, keyword: "ok" }
+          - { index: { } }
+          - { case: "integer_ignored", integer: "not-an-integer", keyword: "ok" }
+          - { index: { } }
+          - { case: "all_ignored", integer: "not-an-integer", keyword: "long-keyword" }
+
+---
+"All fields correct":
+  - do:
+      esql.query:
+        body:
+          query: 'from test metadata _ignored | WHERE case == "ok" | limit 2 | keep integer, keyword, _ignored'
```

Comment:
Nit: either all keywords lower case or upper case (preferred, for consistency); same with queries below.
But can be merged as is and potentially touched at a later time.

Reply:
Ouch, absolutely. Standardizing that file now